### PR TITLE
Display Clair security information

### LINKED ISSUE #32: Display Clair security information.
PierOne now supports Clair for vulnerability scanning. Pierone now exposes the following three information for each tag:

- clair_id
- severity_fix_available
- severity_no_fix_available

The PierOne CLI should now also enhance the `tags` subcommand with these information and provide a new `cves` subcommand to display full CVE reports.

    $ pierone tags foo bar

Team | Artifact | Tag | Created | By | Fixable CVE Severity | Unfixable CVE Severity |
--------|-----------|-------|------------|-----|-----------------------|--------------------------
foo | bar | 1.0 | 5d ago | example | **Critical** | Medium |
foo | bar | 1.1 | 2d ago | example | **Critical** | Medium |
foo | bar | 2.0 | 1d ago | example | None | Medium |

`High` and `Critical` severities should be highlighted.

    $ pierone cves foo bar 1.0

CVE | Severity | Affected Feature | Fixing Feature | Link
-------|------------|------------------------|---------------------|-------
CVE-2014-9471 | Low | coreutils:8.23-4 | coreutils:9.23-5 | https://security-tracker.debian.org/tracker/CVE-2014-9471

Again, `High` and `Critical` needs to be highlighted and the whole table should be sorted by severity. PierOne source contains an ordered list of possible values.

The information for this output can be retrieved via the [Clair API](https://github.com/coreos/clair/blob/master/api/v1/README.md#get-layersname) using the PierOne provided Clair ID. For this, the PierOne CLI will need to learn about the Clair API's endpoint.

### PR DESCRIPTION
**Pull request by rafaelcaricio**
Fixes #32

### CODE REVIEW

**Comment by rafaelcaricio** [comment]
@hjacobs Could you please check this PR?

I just could not figure out how to save the `CLAIR_URL` to pierone configuration file so we do not need to pass `--clair-url https://clair.example.com` all the time in the `pierone cves` command.

File: pierone/cli.py [hunk 1]
```diff
@@ -184,14 +217,76 @@ def tags(config, team: str, artifact, url, output):
...
-        print_table(['team', 'artifact', 'tag', 'created_time', 'created_by'], rows,
-                    titles={'created_time': 'Created', 'created_by': 'By'})
+        titles = {
+            'created_time': 'Created',
+            'created_by': 'By',
+            'severity_fix_available': 'Fixable CVE Severity',
+            'severity_no_fix_available': 'Unfixable CVE Severity'
+        }
+        styles = {
+            'Critical': {'bold': True},
```
Comment (hjacobs) [inline pierone/cli.py]: I would make it red, too

File: pierone/cli.py [hunk 2]
```diff
@@ -184,14 +217,76 @@ def tags(config, team: str, artifact, url, output):
...
+    rows = []
+    token = get_token()
+    for artifact_tag in get_tags(config.get('url'), team, artifact, token):
+        if artifact_tag['name'] == tag:
+            installed_software = get_clair_features(config.get('clair_url'), artifact_tag.get('clair_id'), token)
+            for software_pkg in installed_software:
+                for cve in software_pkg.get('Vulnerabilities', []):
+                    rows.append({
+                        'cve': cve['Name'],
+                        'severity': cve['Severity'],
```
Comment (hjacobs) [inline pierone/cli.py]: I would make severity all upper case here to match all the other outputs we usually have (Senza, etc)

File: pierone/cli.py [hunk 3]
```diff
@@ -184,14 +217,76 @@ def tags(config, team: str, artifact, url, output):
...
+    with OutputFormat(output):
+        titles = {
+            'cve': 'CVE',
+            'severity': 'Severity',
+            'affected_feature': 'Affected Feature',
+            'fixing_feature': 'Fixing Feature',
+            'link': 'Link'
+        }
+        styles = {
+            'Critical': {'bold': True},
```
Comment (hjacobs) [inline pierone/cli.py]: we can have a global styles dict :smile:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**